src/version/108/frontend/nl2go_manager.php

```diff
@@ -145,14 +150,12 @@ public function getCustomers()
 		$subscriber = filter_input(INPUT_POST, 'subscriber', FILTER_VALIDATE_BOOLEAN);
 		$group = filter_input(INPUT_POST, 'group');
 
-		// Include real customers.
-		$customers = $this->getRealCustomers($subscriber);
-
-		// Get subscribed customer only if group is newsletter subscriber group.
-		if ($group === self::NEWSLETTER_SUBSCRIBER_GROUP_ID) {
-			$subscribers = $this->getNewsletterSubscribers();
-			$customers = array_merge($customers, $subscribers);
-		}
+        // Get subscribed customer only if group is newsletter subscriber group.
+        if ($group === self::NEWSLETTER_SUBSCRIBER_GROUP_ID) {
+            $customers = $this->getNewsletterSubscribers();
+        } else {
+            $customers = $this->getRealCustomers($subscriber);
+        }
 
```

Comment:
is this a revert by mistake?

Reply:
I accidentally made changes in v107 that is already out for INT-1087. So i moved those to v108 in [320f7b](/newsletter2go/newsletter2go-jtl3shop-plugin/pull/9/commits/320f7b07fa1ae90584ba63c0fc92a6694bade06d)

Those changes slipped through earlier here: [489dff8](https://github.com/newsletter2go/newsletter2go-jtl3shop-plugin/commit/489dff80d6bbe09b87f180d1e7fdad3919a6a2a4)

So the changes marked are intended.